sdk/eventgrid/eventgrid-systemevents/src/index.ts

```diff
@@ -307,8 +308,8 @@ export {
   ResourceNotificationsResourceDeletedEventData,
   ResourceNotificationsResourceDeletedDetails,
   ResourceNotificationsResourceManagementDeletedEventData,
-  ServiceApiVersions,
-} from "./cadl-generated/index.js";
+  KnownServiceApiVersions
```

Comment:
Looks that previously the generated code might have been modified.  Since this is still beta I decided to keep the generated `KnownServiceApiVersions` enum, which is consistent with other Known* enums.